Taskfile.yml

```
@@ -522,6 +549,22 @@ tasks:
           DATA_DIR: "{{.DATA_DIR}}"
           OUTPUT_FILE: "{{.TMP_CHECKSUM_FILE}}"
       - defer: "rm -f '{{.TMP_CHECKSUM_FILE}}'"
+      # Check that the directory exists and the checksum matches; otherwise delete the checksum file
       - >-
-        diff -q '{{.TMP_CHECKSUM_FILE}}' '{{.CHECKSUM_FILE}}' 2> /dev/null
-        || rm -f '{{.CHECKSUM_FILE}}'
+        (
+        [[ -d "{{.DATA_DIR}}" ]]
```

Is the `[]` syntax equivalent to `test`? Maybe we want to stick with one?
```suggestion
        test -d "{{.DATA_DIR}}"
```

---

It's a bash-specific syntax, so `test` is a better idea.